src/client/options/mod.rs

```diff
@@ -435,6 +435,16 @@ pub enum Tls {
     Disabled,
 }
 
+impl Tls {
+    #[cfg(test)]
+    pub fn options(&self) -> Option<&TlsOptions> {
```

Comment:
Is there a reason you couldn't use `ClientOptions::tls_options` (or modify it to return a reference, if you wanted to avoid the clone)?

Reply:
Oh I didn't see that--yeah that would work nicely here, updated.